DrupalExtended/ruleset.xml

```diff
@@ -0,0 +1,175 @@
+    <exclude name="SlevomatCodingStandard.Classes.ForbiddenPublicProperty"/>
+    <!-- Same as 'Squiz.WhiteSpace.FunctionSpacing.After'. -->
+    <exclude name="SlevomatCodingStandard.Classes.MethodSpacing"/>
+    <!-- Not very useful, because requires spaces even if parent call is the only method contents. -->
+    <exclude name="SlevomatCodingStandard.Classes.ParentCallSpacing"/>
+    <exclude name="SlevomatCodingStandard.Classes.PropertyDeclaration"/>
+    <exclude name="SlevomatCodingStandard.Classes.PropertySpacing"/>
+    <exclude name="SlevomatCodingStandard.Classes.RequireAbstractOrFinal"/>
+    <exclude name="SlevomatCodingStandard.Classes.RequireConstructorPropertyPromotion"/>
+    <!-- Forces for each method, not only constructors. -->
+    <exclude name="SlevomatCodingStandard.Classes.RequireMultiLineMethodSignature"/>
+    <!-- Not very friendly for constructor property promotion with a single param. -->
+    <exclude name="SlevomatCodingStandard.Classes.RequireSingleLineMethodSignature"/>
+    <exclude name="SlevomatCodingStandard.Classes.SuperfluousAbstractClassNaming"/>
+    <exclude name="SlevomatCodingStandard.Classes.SuperfluousErrorNaming"/>
+    <exclude name="SlevomatCodingStandard.Classes.SuperfluousExceptionNaming"/>
+    <exclude name="SlevomatCodingStandard.Classes.SuperfluousInterfaceNaming"/>
+    <exclude name="SlevomatCodingStandard.Classes.SuperfluousTraitNaming"/>
+    <exclude name="SlevomatCodingStandard.Classes.TraitUseDeclaration"/>
+    <exclude name="SlevomatCodingStandard.Classes.TraitUseSpacing"/>
+    <exclude name="SlevomatCodingStandard.Commenting.DeprecatedAnnotationDeclaration"/>
+    <exclude name="SlevomatCodingStandard.Commenting.DisallowCommentAfterCode"/>
+    <!-- 'Drupal.Commenting.DocComment.SpacingBeforeTags' should cover it. -->
+    <exclude name="SlevomatCodingStandard.Commenting.DocCommentSpacing"/>
+    <exclude name="SlevomatCodingStandard.Commenting.ForbiddenAnnotations"/>
+    <exclude name="SlevomatCodingStandard.Commenting.ForbiddenComments"/>
+    <exclude name="SlevomatCodingStandard.Commenting.InlineDocCommentDeclaration"/>
+    <exclude name="SlevomatCodingStandard.Commenting.RequireOneLineDocComment"/>
+    <exclude name="SlevomatCodingStandard.Commenting.RequireOneLinePropertyDocComment"/>
+    <!-- Conflicts with Drupal coding standards. -->
+    <exclude name="SlevomatCodingStandard.Commenting.UselessInheritDocComment"/>
+    <exclude name="SlevomatCodingStandard.Complexity.Cognitive"/>
+    <exclude name="SlevomatCodingStandard.ControlStructures.AssignmentInCondition"/>
+    <exclude name="SlevomatCodingStandard.ControlStructures.BlockControlStructureSpacing"/>
+    <exclude name="SlevomatCodingStandard.ControlStructures.DisallowNullSafeObjectOperator"/>
+    <exclude name="SlevomatCodingStandard.ControlStructures.DisallowShortTernaryOperator"/>
+    <exclude name="SlevomatCodingStandard.ControlStructures.DisallowYodaComparison"/>
```

Comment:
Also, nice to have by default. There is no reason to mix styles. By default, Drupal doesn't use Yoda Comparison. It should be safe to enable it.